demos/course/chapters/00-account.html

```diff
@@ -7,14 +7,36 @@
     <meta name="viewport" content="width=device-width, initial-scale=1"> 
     <title>Account - How to Make French Toast - xapi-jqm</title> 
     <link rel="stylesheet" href="http://code.jquery.com/mobile/1.4.2/jquery.mobile-1.4.2.min.css" />
+    
+    <!-- Load CDN; if it fails, load the local instance. -->
     <script src="http://code.jquery.com/jquery-1.10.2.min.js"></script>
+    <script>
+        if (typeof jQuery == 'undefined')
+        {
+            document.write(unescape("%3Cscript src='/js/jquery-1.10.2.min.js' type='text/javascript'%3E%3C/script%3E"));
+        }
+    </script>
+    
+    <!-- Load CDN; if it fails, load the local instance. -->
     <script src="http://code.jquery.com/mobile/1.4.2/jquery.mobile-1.4.2.min.js"></script>
+    <script>
+        if(typeof $.mobile == 'undefined')
+        {
+            document.write(unescape("%3Cscript src='/js/jquery.mobile-1.4.2.min.js' type='text/javascript'%3E%3C/script%3E"));
+        }
+        $(function() {
+            if($('.ui-helper-hidden:first').is(':visible') === true){
+                $('<link rel="stylesheet" type="text/css" href="/css/jquery.mobile-1.4.2.min.css" />').appendTo('head');
```

Comment:
href="/css/jquery.mobile-1.4.2.min.css" - add a '.' in front so it's a relative path else it'll never be found (same for the other other files)

also it's load the css twice (from the cdn and local file) instead of just once